docs/Examples.md

```diff
@@ -261,17 +277,17 @@ metadata:
 spec:
   clusterMasterRef:
     name: cm
+  monitoringConsoleRef:
+    name: example_mc
 EOF
 ```
 
-This will automatically create a deployer with 3 search heads clustered
-together (search head clusters require a minimum of 3 members):
+This will automatically create a deployer with 3 search heads clustered together. Search head clusters require a minimum of 3 members.
 
 ```
 $ kubectl get pods
 NAME                                        READY   STATUS    RESTARTS   AGE
 splunk-cm-cluster-master-0                   1/1     Running   0          53m
-splunk-default-monitoring-console-0          0/1     Running   0          52m
 splunk-example-deployer-0                    0/1     Running   0          29s
```

Comment:
Since we have used the `monitoringConsoleRef` above, don't we still need to show a MC pod?

Reply:
No, unless we show that MC pod is also being created

Reply:
Sorry, but isn't it the point that for verification steps (get pods) that the reader has to have done SOMETHING? After all, if they've done nothing than there won't be any pods created and therefore nothing will be returned by the get pods command. So please choose what assumptions you'd like to make, and let's stick with them. Does the reader do all the things, some of the things, or none of the things? If some, please state exactly which ones we're assuming they've run.
I hope that puts perspective around the goals here. Please reply with your chosen assumptions, and the examples can be adjusted accordingly.

Reply:
I see your point. Here is my take on this:

* I agree with the earlier point made about not having a preference wrt this: `There is no preferred order when running a Monitoring Console pod; you can start the pod before or after the other CR's in the namespace`
* Having said that it is strongly encouraged that customers do create the MC pod (earlier or later), so that they take advantage of the MC functionality
* With the above in mind, I am inclined towards our examples listing the mc pod, and hence giving the message to customers that the MC pod's existence is expected
* From verification steps perspective, I am thinking a one liner added each section would help? Something like `You can create a search head cluster that is configured to communicate with your indexer cluster by using a SearchHeadCluster resource
and adding the clusterMasterRef parameter. Additionally, monitoringConsoleRef can also be added to the Spec to reference the Monitoring Console pod (which is either pre-existing or to be created). `